src/Routing/ResourceRegistrar.php

```diff
@@ -31,10 +31,13 @@ class ResourceRegistrar
 {
 
     const KEYWORD_RELATIONSHIPS = 'relationships';
+    const KEYWORD_PROCESSES = 'queue-jobs';
```

Comment:
If making configurable, then need to check uses of this constant as it would have to use the configured name instead of the constant (or keep the constant as the default to use if none configured).